README.md

```diff
@@ -57,3 +57,34 @@ configuration.setVersion("v1");
 // deploy
 deployment.deploy(deployConfiguration);
 ```
+
+## SDK Manager/Downloader
```

Comment:
can you just call it the SDK Manager? downloading is part of what the manager does. I don't really care though.

Reply:
yeah